politeiawww/proposals.go

```diff
@@ -200,6 +208,28 @@ func (p *politeiawww) linkByPeriodMax() int64 {
 	return 7776000 // 3 months in seconds
 }
 
+func (p *politeiawww) linkByValidate(linkBy int64) error {
```

Comment:
Can we maybe name this to `validateLinkBy` so that it follows the same format as the other validation functions? Thanks mate.

Reply:
If you start the function name by referencing the object (`linkBy`) instead of the action (`validate`) it allows you to group the functions together and makes them easier to find.  If I'm writing code that does something with the linkby and want to see what functions have already been written, its much easier if they're grouped by the object name since the action names are likely to be far less consistent and open to interpretation (ex. `validate`, `verify`, `check`).  I can just pull up the list of function declarations and look for all of the `linkBy` functions to see if what I'm trying to do already exists.
```
linkByPeriodMin() int64
linkByPeriodMax() int64
linkByValidate(linkBy int64)
```
Compared to:
```
minLinkByPeriod() int64
maxLinkByPeriod() int64
validateLinkBy(linkBy int64)
```

Normally I would agree that you should always match the existing style of the codebase you're contributing to, but in this situation I want to move the politeiawww style more towards what I listed above.   In short, it makes it easier to organize code and find existing functions.

Reply:
Makes perfect sense, appreciate your explanation on this. Agreed that way is better and more consistent. Will keep this in mind for future reviews/implementations, so that we can move towards the style described above.